### PR TITLE
Blogpost

### PR DESCRIPTION
**Pull request by valeriafurin**
Hi there, 
I wrote the Blog Component with testing. 
In the next PR I am going to add additional functionality to this component.

Thanks

### CODE REVIEW

File: src/components/Menu/Menu.tsx [hunk 1]
```diff
@@ -10,18 +10,22 @@ export const Menu: FunctionComponent<MenuProps> = ({ open }) => {
 
   return (
     <StyledMenu open={open} aria-hidden={!open}>
-      <a href="/" tabIndex={tabIndex}>
+      <a href="/About" tabIndex={tabIndex}>
```
Comment (barsukov) [inline src/components/Menu/Menu.tsx:13]: Could you please use Link component from React rotuer for this links in your nav bar?

File: src/pages/AboutPage.test.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
```
Comment (barsukov) [inline src/pages/AboutPage.test.tsx:1]: could you also pleasw move this file under the __test__ folder

**Review by barsukov — CHANGES_REQUESTED** [review]
Nice PR I have a few comments

File: src/components/Menu/Menu.tsx (hunk 1 above)
Comment (valeriafurin) [reply src/components/Menu/Menu.tsx]: Done 👍

**Review by barsukov — APPROVED** [review]
👍

File: src/pages/AboutPage.test.tsx (hunk 2 above)
Comment (valeriafurin) [reply src/pages/AboutPage.test.tsx]: Done 👍

**Pull request merged**